cdap-ui/app/directives/widget-container/widget-toggle-switch/widget-toggle-switch.less

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import '../../../styles/variables.less';
+
+.toggle-switch {
+  height: 32px;
+  width: 65px;
+  position: relative;
+  vertical-align: middle;
+  border-radius: 5px;
+  font-weight: 500;
+  color: white;
+  user-select: none;
+  cursor: pointer;
+  display: inline-block;
+
+  .switch-button {
+    position: absolute;
+    top: 2px;
+    display: block;
+    width: 14px;
+    height: 28px;
+    background-color: white;
+    border-radius: 5px;
+    transition: all 0.3s ease-in 0s;
```

Comment:
Don't think this is being applied right now. We need to specify animation keyframes for this to kick in right?